src/schema/actions.ts

```diff
@@ -82,9 +82,12 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   Query: {
     actions: (_, __, ctx: AuthContext, info): Promise<GQLUserAction[]> =>
       graphorm.query<GQLUserAction>(ctx, info, (builder) => {
-        builder.queryBuilder = builder.queryBuilder.where({
-          userId: ctx.userId,
-        });
+        builder.queryBuilder = builder.queryBuilder.where(
```

Comment:
actions might be tricky because for some we rely to not show popups when action is done, so people could see popups multiple times if read replica does not get the action update